xdocs/usermanual/component_reference.xml

```diff
@@ -6521,17 +6521,39 @@ See <a href="get-started.html#classpath">JMeter's Classpath</a> and
        The users are generated according to the schedule.
     </p>
     <p>The load profile consists of a sequence of constant, increasing or decreasing load.
-       The following commands are available:
+       The basic configuration is <code>rate(1/sec) random_arrivals(2 min) rate(3/sec)</code> which means the load will increase linearly
+        from 1 request per second to 3 requests per second during 2-minute timeframe.
+       If you omit "end" rate, then it will be set to the same value as "start". For example,
+        <code>rate(1/sec) random_arrivals(2 min)</code> is exactly the same as <code>rate(1/sec) random_arrivals(2 min) rate(1/sec)</code>.
+       That is why <code>rate(1/sec) random_arrivals(2 min) random_arrivals(3 min) rate(4/sec)</code> is exactly the same as
+        <code>rate(1/sec) random_arrivals(2 min) rate(1/sec) random_arrivals(3 min) rate(4/sec)</code>, so the load is 1/sec during the first two minutes,
+        and then it increases linearly from 1/sec to 4/sec during 3 minutes.
+    </p>
+    <p>Here are examples for using the schedule:
+        <ul>
+            <li><code>rate(10/sec) random_arrivals(1 min) rate(10/sec)</code> &mdash; constant load rate of 10/sec during 1 minute</li>
```

Comment:
We could use a definition list instead of an unordered list here.
Plus the usual stuff, I wrote above. Numbers spelled out and highlighting of the given rates.